solutions/standard/main.tf

```diff
@@ -194,3 +191,70 @@ module "event_notifications" {
   skip_en_cos_auth_policy = var.skip_en_cos_auth_policy
   cos_endpoint            = local.cos_endpoint
 }
+
+# create a service authorization between Secrets Manager and the target service (Event Notification)
+resource "ibm_iam_authorization_policy" "secrets_manager_key_manager" {
+  count                       = var.skip_en_sm_auth_policy || var.existing_secrets_manager_instance_crn == null ? 0 : 1
+  depends_on                  = [module.event_notifications]
+  source_service_name         = "secrets-manager"
+  source_resource_instance_id = local.existing_secrets_manager_instance_guid
+  target_service_name         = "event-notifications"
+  target_resource_instance_id = local.eventnotification_guid
+  roles                       = ["Key Manager"]
+  description                 = "Allow Secrets Manager with instance id ${local.existing_secrets_manager_instance_guid} to manage key for the event-notification instance"
```

Comment:
Is there enough allowed character to also include the event notification guid in the description?

Reply:
@Aashiq-J, @ocofaigh, I won't be able to review this PR. Sorry.